src/guides/v2.4/graphql/mutations/remove-requisition-list-items.md

```diff
@@ -0,0 +1,83 @@
+---
+group: graphql
+title: removeRequisitionListItems mutation
```

Comment:
This is now named deleteRequisitionListItems and the schema is now very different.